package.json

```diff
@@ -75,7 +76,9 @@
     "Friedel Ziegelmayer <dignifiedquire@gmail.com>",
     "Greenkeeper <support@greenkeeper.io>",
     "Richard Littauer <richard.littauer@gmail.com>",
+    "Tom Swindell <t.swindell@rubyx.co.uk>",
     "Yusef Napora <yusef@napora.org>",
+    "dryajov <dryajov@gmail.com>",
```

Comment:
Can you drop these two contributor additions, they are auto generated on release